### ISSUE TITLE
BUG: Registered user via service cannot login

### ISSUE THREAD

**Issue by DarthHater** [DarthHater/bored-board-service#22]
Right now I'm wiring up Register to the service in: https://github.com/DarthHater/bored-board-webapp/pull/23

I am able to register the user, but after registering when I attempt to login I am getting:

```
web_1       | time="2018-07-09T16:47:02Z" level=error msg="Wrong password" username=FefflonDon
web_1       | time="2018-07-09T16:47:02Z" level=warning msg="172.20.0.1 - 07e61ebf0454 [09/Jul/2018:16:47:02 +0000] \"POST /login\" 401 24 \"http://0.0.0.0:8090/\" \"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36\" (51389ms)" clientIP=172.20.0.1 dataLength=24 hostname=07e61ebf0454 latency=51389 method=POST path=/login referer="http://0.0.0.0:8090/" statusCode=401 userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36"
```

There appears to be some mismatch in encryption or etc...

@andysbolton if you want to take a gander at something, this would be a good issue!

**Comment by DarthHater** [comment]
Suggestion on debugging this is to examine the difference between passwords created by psql and then via the service, and decode etc... compare them.

**Comment by DarthHater** [comment]
This turned out to be an issue with the frontend, and passing `password` instead of `userpassword` to the backend. I think what we could do here is standardize on using `Password` in our structs, so we don't have a lot of different ways of saying the same thing. @andysbolton wdyt?

**Comment by andysbolton** [comment]
Yeah, I think that's a good idea to rename all password fields in structs to `Password`.

**Comment by andysbolton** [comment]
This should be addressed in #21 -- all struct fields have been renamed to `Password`.
